crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -110,6 +110,126 @@ impl NodeId {
     pub fn is_dummy(&self) -> bool {
         self.0 == SsaContext::dummy_id()
     }
+
+    /// Create a NodeId to represent a constant value
+    /// TODO: change this to from_const, so it reads NodeId::from_const
```

Comment:
I don't know if `from_const` would be the best name here. To me that doesn't convey that it is inserting a const value into the context.

Reply:
Yeah that makes sense, If I can't think of something better might just keep it the way it is :y